packages/rocketchat-lib/server/functions/createRoom.js

```diff
@@ -86,7 +86,7 @@ RocketChat.createRoom = function(type, name, owner, members, readOnly, extraData
 
 	if (type === 'c') {
 		Meteor.defer(() => {
-			RocketChat.callbacks.run('afterCreateChannel', owner, room);
+			RocketChat.callbacks.run('afterCreateChannel', room);
```

Comment:
Please revert this and instead change the piwik package, as there are incoming changes which rely on these arguments. See my pull request #5823